File: dakobed-algorithms/python-algorithms/arrays/twoSum.py
class Solution:
    def twoSum(self, nums, target):
        complements = dict()
        for index, num in enumerate(nums):
            if num in complements:
                return [index, complements[num]]
            complements[target-num] = index

nums = [2,7,11,15]
target = 9

solution = Solution()
solution.twoSum(nums, target)

File: dakobed-algorithms/java-algorithms/src/main/java/arrays/twoSum/twoSum.java
package arrays.twoSum;

public class twoSum {
    public static void main(String[] args) {
        int[] nums = {2,7,11,15};
        int target = 9;
        Solution solution = new Solution();
        int[] indices = solution.twoSum(nums, target);
        for(int num: indices){
            System.out.println(num);
        }
    }
}
